form/src/main/java/org/dhis2/form/ui/provider/inputfield/FieldProvider.kt

```diff
@@ -79,14 +80,16 @@ fun FieldProvider(
         .onSizeChanged { intSize ->
             visibleArea = Rect(
                 size = Size(intSize.width.toFloat(), intSize.height.toFloat()),
-                offset = Offset(0f, 20f),
+                offset = Offset(0f, 200f),
             )
         }
         .onFocusChanged {
             if (it.isFocused && !fieldUiModel.focused) {
                 scope.launch {
-                    bringIntoViewRequester.bringIntoView(visibleArea)
                     fieldUiModel.onItemClick()
+
+                    delay(10)
```

Comment:
Is this delay blocking the main thread?

Reply:
This suspends the coroutine but does not block the main thread

Reply:
The coroutine starts on the main thread.
When delay(10) is called, the coroutine is suspended for 10 milliseconds.
The main thread is free to perform other tasks during this delay period.
After 10 milliseconds, the coroutine resumes execution on the main thread.